dask/tests/test_base.py

```diff
@@ -955,6 +955,31 @@ def test_visualize():
         assert os.path.exists(os.path.join(d, "mydask.png"))
 
 
+@pytest.mark.skipif("not da")
+@pytest.mark.skipif(
+    sys.flags.optimize, reason="graphviz exception with Python -OO flag"
+)
+def test_visualize_no_filename(monkeypatch):
+    pytest.importorskip("graphviz")
+    pytest.importorskip("dask.dot")
+
+    x = da.arange(10)
+
+    test_list = []
```

Comment:
For clarity, you might like to rename `test_list` to something more meaningful. Maybe something like `was_mock_function_called`?